examples/hello-world/Makefile.c64.defines

```diff
@@ -0,0 +1,2 @@
+DEFINES = WITH_CLIENT,WITH_DNS,WITH_PFS,WITH_LOGGING
+CONTIKI_WITH_IPV4 = 1
```

Comment:
I had a look on savedefines plus README-BUILDING.md and it mentions:

``` 
Beside TARGET= there's DEFINES= which allows to set arbitrary variables for the
C preprocessor in form of a comma-separated list. Again it is possible to avoid
having to re-type i.e. DEFINES=MYTRACE,MYVALUE=4711 by running make TARGET=esb
DEFINES=MYTRACE,MYVALUE=4711 savedefines. A file called Makefile.esb.defines is
saved in the project's directory containing the currently saved defines for the
ESB platform.
```
The savedefines build receipe just saves DEFINE provided by the make invocation.

Thus 
```
make  TARGET=c64 DEFINES=WITH_LOGGING"\n"CONTIKI_WITH_IPV4=1  savedefines
```
would also deliver the above commented lines.

O.K. this is a little bit far-fetched, so you are right ... also doesn't seem valid to me now.

One problem I got with the current platform implementation for c64 is, that it assumes that the user wants to use it on a c64 with ethernet interface. The hello-world example does not require any ethernet support in my opinion.

Thus I recommend to add another define "WITH_ETH" or similar, to be able to enable the ethernet support explicitly.

I am quite  sure that this problem does not apply to platform c64  only ...

Regarding "CONTIKI_WITH_IPV4 = 1" : I will try to fix this one and hope that it does not break other platforms.